src/libtorch.cc

```diff
@@ -552,11 +582,48 @@ ModelInstanceState::Create(
 
 ModelInstanceState::ModelInstanceState(
     ModelState* model_state, TRITONBACKEND_ModelInstance* triton_model_instance)
+#ifdef TRITON_ENABLE_GPU
+    : BackendModelInstance(model_state, triton_model_instance),
+      model_state_(model_state), device_(torch::kCPU), is_dict_input_(false),
+      pytorch_stream_(at::cuda::getDefaultCUDAStream())
+#else
     : BackendModelInstance(model_state, triton_model_instance),
       model_state_(model_state), device_(torch::kCPU), is_dict_input_(false)
+#endif
 {
   if (Kind() == TRITONSERVER_INSTANCEGROUPKIND_GPU) {
     device_ = torch::Device(torch::kCUDA, DeviceId());
```

Comment:
Should this be also in the ENABLE_GPU condition?

Reply:
Fixed.